user-docs/latest/content-manager/introduction-to-content-manager.md

```diff
@@ -0,0 +1,100 @@
+# Introduction to the Content Manager
+
+The Content Manager is a core plugin of Strapi. It is a feature that is always activated by default and cannot be deleted. It is accessible both when the application is in a development and production environment.
+
+The Content Manager is divided into 2 categories, both displayed in the main navigation: *Collection types* and *Single types*. Each category contains the available collection and single content-types, which were created beforehand using the Content-Types Builder. From these 2 categories, admin panel users can create, manage, and publish content.
+
+::: tip 💡 TIP
+Click the search icon <Fa-Search /> in the main navigation to use a text search and find one of you content types more quickly!
+:::
+
+## Collection types
```

Comment:
I saw in the "Deleting content" section of the "Saving, publishing and deleting content" page
you added a not about the trash button and bulk delete action.
I was questioning if it's not interesting to have also this information or bulk actions, edit button, trash button of an entry line in this section.

Reply:
I think I'll keep it this way for now, as I feel like the structure is good and content was approved. Perhaps we'll move things around in the future but since we need to finish this documentation for the end of the week, it's better to stop there and improve after, once we'll be able to take a step back and re-discover this documentation with new eyes 🤓